sdk/servicebus/azure-servicebus/stress/scripts/stress_test_base.py

```diff
@@ -276,20 +288,29 @@ def _receive(self, receiver, end_time):
             raise
 
     def run(self):
+
         start_time = datetime.utcnow()
+        if isinstance(self.duration, int):
+            self.duration = timedelta(seconds=self.duration)
         end_time = start_time + (self._duration_override or self.duration)
+
         with self.process_monitor:
             with concurrent.futures.ThreadPoolExecutor(max_workers=4) as proc_pool:
```

Comment:
how many senders and or receivers can there be? You could end up in a situation where there are 3 senders, 2 receivers and the last receiver never runs as it has to wait for a free thread.

Reply:
all test right now are 1:1